docs/using-the-compiler.rst

```diff
@@ -378,12 +378,11 @@ Input Description
         {
           // Choose which model checker engine to use: all (default), bmc, chc, none.
           "engine": "chc",
-          // Choose which targets should be checked: all (default), constantCondition,
+          // Choose which targets should be checked: constantCondition,
           // underflow, overflow, divByZero, balance, assert, popEmptyArray, outOfBounds.
+          // If the option is not given all targets are checked by default.
           // See the Formal Verification section for the targets description.
-          // Multiple targets can be selected at the same time, separated by a comma
-          // without spaces:
-          "targets": "underflow,overflow,assert",
```

Comment:
When was this option introduced? One or two releases ago?

Reply:
0.8.1